tests/e2e/mvp_test.go

```diff
@@ -42,7 +55,53 @@ func TestMVP(t *testing.T) {
 	// then the max cap limit is persisted
 	q := baseapp.QueryServiceTestHelper{GRPCQueryRouter: consumerApp.GRPCQueryRouter(), Ctx: consumerChain.GetContext()}
 	var rsp types.QueryVirtualStakingMaxCapResponse
-	err := q.Invoke(nil, "/osmosis.meshsecurity.v1beta1.Query/VirtualStakingMaxCap", &types.QueryVirtualStakingMaxCapRequest{Address: randomContract.String()}, &rsp)
+	err := q.Invoke(nil, "/osmosis.meshsecurity.v1beta1.Query/VirtualStakingMaxCap", &types.QueryVirtualStakingMaxCapRequest{Address: stakingContract.String()}, &rsp)
 	require.NoError(t, err)
 	assert.Equal(t, sdk.NewInt64Coin(sdk.DefaultBondDenom, 1_000_000_000), rsp.Limit)
+
+	// when staking contract is instructed to bond tokens
```

Comment:
Very cool series of tests to give a nice full-stack run-through of at least the basic happy path.